suresh/array-str-methods.js

```diff
@@ -0,0 +1,97 @@
+var array = ['a', 1, 2, 'b', 'xyz'];
+
+function indexOf(array, ele) {
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === ele) {
+      return i;
+    } 
+  }
+  return "element is not present"
```

Comment:
incorrect return. REMEMBER IT MUST ALWAYS RETURN THE **_SAME DATA TYPE_** or null. 

- what data type should this return? Yes! answer it. not rhetorical.